### DISCUSSION TITLE
Getting bounds while ignoring nodata

### DISCUSSION THREAD

**Discussion by glaroc** [developmentseed/titiler#674] [General]
Just a thought: it would be nice to add the possibility to optionally obtain bounds from /cog/bounds or /stac/bounds that ignore nodata values. In other words get bounds for just the portion of the raster that has values in it.

**Comment by vincentsarago** [comment]
@glaroc do you want the `shape` of the data instead of the bounds? 

It should be quite easy to create a `/shape` extension for COG using methods described in https://www.element84.com/blog/the-stactools-raster-footprint-utility

**Reply by glaroc** [reply]
I'd be happy with just the bounds for the moment, but the shape would be even better.
